sources/editor/Stride.RestoreHelper/Stride.RestoreHelper.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
+    <RestoreSources>
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json;
+      https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json;
+      https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json;
+      https://api.nuget.org/v3/index.json
+    </RestoreSources>
+    <NoWarn>$(NoWarn);NU1603;NU1605</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.LanguageServer.Protocol" Version="$(RoslynPrivateVersion)" GeneratePathProperty="true" IncludeAssets="none" PrivateAssets="all" />
+  </ItemGroup>
+  <Target Name="GetLibReferences" Outputs="@(Reference)">
+    <ItemGroup>
+      <Reference Include="$(PkgMicrosoft_CodeAnalysis_LanguageServer_Protocol)/lib/net7.0/*.dll" />
+    </ItemGroup>
+  </Target>
```

Comment:
How does it work? Where is the reference (documentation) for this trick?

Reply:
And one more here: https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty